config.yaml

```diff
@@ -47,5 +47,7 @@ params:
           link: /about/contribution-guide/
         - text: Code of Conduct
           link: /about/coc/
+        - text: Talks
```

Comment:
@ydcjeff I propose to put "Talks" into "About", what do you think ?

Reply:
Sounds good.